lib/fetch/headers.js

```diff
@@ -492,48 +492,33 @@ class Headers {
   keys () {
     webidl.brandCheck(this, Headers)
 
-    if (this[kGuard] === 'immutable') {
-      const value = this[kHeadersSortedMap]
-      return makeIterator(() => value, 'Headers',
-        'key')
-    }
```

Comment:
Why remove this?

Reply:
I don't think it's necessary to keep it, because it's a fast-path to avoid shallow cloning.
However, It can be left to avoid getter.